Identity/Identity.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Identity.swift
+//  RealifeTech
+//
+//  Created by Ross Patman Work on 03/12/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import Foundation
+
+public protocol Identity {
```

Comment:
I have implemented `Identity` as part of https://livestyled.atlassian.net/browse/MOD-797 as well. So I guess I will have to wait till you merge this, otherwise, there will be a lot of conflicts. That means this ticket will be blocking me to finish MOD-797. For me the protocol looks like this:
```
public protocol Identity {
    var sso: SSOProvidable { get }
}

public protocol SSOProvidable {
    func getMyUserSSO(callback: @escaping (Result<String, Error>) -> Void)
}
```